lib/utils/config/src/test/buffer-config-test.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const { bufferConfig } = require('../lib/buffer-config');
+
+describe('buffer config tests', () => {
```

Comment:
These tests do not validates that the returned results are actually the expected